native/packages/react-native-bpk-component-button/src/BpkButton-styles.js

```diff
@@ -128,11 +128,19 @@ const modifiers = {
     container: {
       width: tokens.spacingXl,
     },
+    button: {
+      paddingLeft: 0,
+      paddingRight: 0,
+    },
   }),
   iconOnlyLarge: StyleSheet.create({
     container: {
       width: largeHeight,
     },
+    button: {
+      paddingLeft: 0,
```

Comment:
`paddingHorizontal` could be used here right?

Reply:
Just tried it and it didn't have the same effect, despite what the docs said 😢

Reply:
Strange

Reply:
What happens instead?

Reply:
Ah. I'm being dumb. Because of the cascade, it was inheriting `paddingLeft` and `paddingRight` from another style. I've changed everywhere to use `paddingHorizontal` and `paddingVertical` and it works like a charm now!